assets/onboarding/features/index.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import QueryStringRouter from '../query-string-router';
+import Features from './index';
+
+// TODO: Needs more test after the integration
```

Comment:
Couldn't we test the basic flows now? (Showing modal, moving to final page, moving back to initial page, etc)

Reply:
Added flow tests here: https://github.com/Automattic/sensei/pull/3121/commits/4b8d7fcaff46fcd147e580068bf8da09e52cad75